services/app/src/static/style/antd-override.css

```diff
@@ -1,7 +1,55 @@
+/*Buttons*/
+.ant-btn-primary:disabled:hover {
+  background-color: rgba(0, 0, 0, 0.04) !important;
```

Comment:
Please check antd `ConfigProvider` and make antd use our primary and secondary colors